pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -119,6 +120,11 @@ type HyperConvergedSpec struct {
 	// +kubebuilder:validation:Enum=RemoveWorkloads;BlockUninstallIfWorkloadsExist
 	// +optional
 	UninstallStrategy *HyperConvergedUninstallStrategy `json:"uninstallStrategy,omitempty"`
+
+	// LogVerbosityConfig configures the verbosity level of Kubevirt's different components. The higher
+	// the value - the higher the log verbosity.
```

Comment:
I'm sure there are limitations. I guess no more than 5?

Reply:
No, there is no clear limitation. This can be different for every component and can change in the future.